### PR TITLE
Added responsive hamburger navbar

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. Added responsive hamburger menu

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
       <header class="navbar navbar-home w-full"> 
         <nav class="navbar-wrapper">
           <div class="navbar-left-aligned">
-            <!-- <svg
+            <svg
```
Comment (anupkrbid) [inline docs.html:36]: you can save this SVG in your assets and use it in the HTML as img tag.

File: style.css [hunk 2]
```diff
@@ -48,7 +48,7 @@
   text-align: center;
 }
 
-@media screen and (max-width: 640px){
+@media screen and (max-width: 740px){
```
Comment (anupkrbid) [inline style.css:51]: You should use rems instead of px, px won't work properly when you are using browser zoom.

**Review by anupkrbid — APPROVED** [review]
A few suggestions but overall 👍

**Review by xii-vi — APPROVED** [review]
_Everything is in place. You can merge._

**Pull request merged**